locales-src/onboarding.ftl

```diff
@@ -31,12 +31,20 @@ onboarding-join-form-email =
 onboarding-join-form-email-error = Valid email required
 onboarding-join-form-legal = By proceeding, you agree to the <a data-l10n-name="terms">Terms of Service</a> and <a data-l10n-name="privacy">Privacy Notice</a>.
 onboarding-join-form-continue = Continue
+onboarding-join-form-signin-label = Already have an account?
+onboarding-join-form-signin = Sign In
 
 onboarding-start-browsing-button-label = Start Browsing
 onboarding-cards-dismiss =
     .title = Dismiss
     .aria-label = Dismiss
 
+## Welcome full page string
+
+onboarding-fullpage-welcome-subheader = Let’s start exploring everything you can do.
+onboarding-fullpage-form-email =
+    .placeholder = Your email address...
```

Comment:
You should use `…` (single unicode character here)

Reply:
done